docs/excel/excel-add-ins-multiple-ranges.md

```diff
@@ -0,0 +1,208 @@
+    const formulaLogicalNumberRanges = usedRange.getSpecialCells("Formulas", "LogicalNumbers");
+    formulaLogicalNumberRanges.format.fill.color = "pink";
+
+    return context.sync();
+})
+```
+
+> [!NOTE]
+> The `Excel.SpecialCellValueType` parameter can only be used if the `Excel.SpecialCellType` parameter is "Formulas" or "Constants".
+
+### Getting RangeAreas within RangeAreas
+
+The `RangeAreas` type itself also has `getSpecialCells` and `getSpecialCellsOrNullObject` methods which take the same two parameters. These methods return all the targeted cells from all of the ranges in the `RangeAreas.areas` collection. There is one small difference in the behavior of the methods when called on a `RangeAreas` object instead of a `Range` object: when you pass "SameConditionalFormat" as the first parameter, the method returns all cells that have the same conditional formatting as the upper leftmost cell *in the first range in the `RangeAreas.areas` collection*. The same point applies to "SameDataValidation": when passed to `Range.getSpecialCells`, it returns all cells that have the same data validation rule as the upper leftmost cell *in the range*; but when it is passed to `RangeAreas.getSpecialCells`, it returns all cells that have the same data validation rule as the upper leftmost cell *in the first range in the `RangeAreas.areas` collection*.
+
+## Reading properties of RangeAreas
+
+Reading property values of `RangeAreas` requires care, because a given property may have different values for different ranges within the `RangeAreas`. The general rule is that if a consistent value *can* be returned it will be returned. For example, in the following code, The RGB code for pink (`#FFC0CB`) and `true` will be logged to the console because both the ranges in the `RangeAreas` object have a pink fill and both are entire columns.
+
+```js
+Excel.run(function (context) {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+
+    // The ranges are the F column and the H column.
+    const rangeAreas = sheet.getRanges("F:F, H:H");  
+    rangeAreas.format.fill.color = "pink";
+
+    rangeAreas.load("format/fill/color, isEntireColumn");
+
+    return context.sync()
+        .then(function () {
+            console.log(rangeAreas.format.fill.color); // #FFC0CB
+            console.log(rangeAreas.isEntireColumn); // true
+        })
+        .then(context.sync);
+})
+```
+
+Things get more complicated when consistency isn't possible. For example, the following code creates a `RangeAreas` in which only one range is an entire column and only one is filled with pink. The console will show `null` for the fill color,  `false` for the `isEntireRow` property, and "Sheet1!F3:F5, Sheet!H:H" (assuming the sheet name is "Sheet1") for the `address` property. This illustrates three principles:
+
+- A boolean property of a `RangeAreas` object returns `false` unless the property is true for all the member ranges.
```

Comment:
Consider leading with the principles of property returns, then discussing the example.